kinappserver/models/offer.py

```diff
@@ -258,3 +259,40 @@ def get_offer_details(offer_id):
         return {'title': 'unknown offer', 'desc': '', 'provider': {}}
 
     return {'title': offer.title, 'desc': offer.desc, 'provider': offer.provider_data}
+
+
+def set_locked_offers(user_id, days):
+    """ scan the db for offers bought in the last {days} and update USER_LOCKED_OFFERS_REDIS_KEY"""
+    from datetime import datetime, timedelta
```

Comment:
There is no reason to import it everytime you start the method. Just import it at the top if its not a one-time thing.